.github/workflows/aggregator-stress-test.yml

```diff
@@ -2,6 +2,19 @@ name: Aggregator stress test
 
 on:
   workflow_call:
```

Comment:
Should we swap the order of `workflow_call` and `workflow_dispatch` to have the dispatch first since it's the one that specify the description ?
_same comment for the two other workflows_

Reply:
Yes, good idea. I switched the order :+1: